### ISSUE TITLE
Colors

### ISSUE THREAD

**Issue by hoelzer** [Psy-Fer/CoVarPlot#6]
The cyan and the pink might be difficult to distinguish for color blind people. No problem for me, but a colleague just told me that it's hard for him to distinguish the colors. 

It should be easy to change the two colors to something color-blind-friendly? I think there are recommended color palettes for such an purpose.

**Comment by Psy-Fer** [comment]
Thanks for the feedback!

I actually changed them to this to make it colourblind friendly (had some CB people say it was alright).

But, maybe I'm not covering the type your colleague has.
Do they have a colour combo that works well for them? I'd happily change it, or even just add in some CB flags which change the colours to different presets. 

Let me know?

James

**Comment by hoelzer** [comment]
Hah interesting :) I will ask him what kind of colourblindness he has and which colours he would prefer

**Comment by hoelzer** [comment]
Okay, he went and looked at it on the computer instead of the phone and actually, the colors are not so bad and he can distinguish.

However, color-blindness wise pink and cyan might not be the most lovely combination?

ColorBrewer palettes are always good for this, you can select one and check if they are color blindness friendly: https://colorbrewer2.org/#type=qualitative&scheme=Dark2&n=3 for example

Also a good scheme: https://colorbrewer2.org/#type=qualitative&scheme=Paired&n=3

So in short: seems the first impression was not so good bc/ he also looked from the phone but maybe you also like some of the color palettes posted here that might work better?

**Comment by Psy-Fer** [comment]
Ooh that first one is good, even from a non colourblindness point of view. 

What i'm going to do, is add an argument which has a number of choices. Each choice will be suitable for the 3 or so kinds of colourblindness.

I'll then use this same method in all future tools.

I'll get around to it this week at some point.

Thanks again!
